hawk/app/models/crm_config.rb

```diff
@@ -57,7 +57,10 @@ def mapping
             [
               "pengine",
               "crmd",
```

Comment:
According to https://clusterlabs.org/pacemaker/doc/en-US/Pacemaker/2.0/html/Pacemaker_Explained/_pacemaker_architecture.html, crmd should be changed to pacemaker-controld. Can you confirm this @krig ?

Reply:
yeah, that's why i added pacemaker-controld to the list below - still keeping crmd on the list in case hawk is running together with an older version of pacemaker.